olive/passes/olive_pass.py

```diff
@@ -246,7 +246,10 @@ def run(
             # assumption: the model attributes from passes, if any, are more important than
             # the input model attributes, we should not update/extend anymore outside of the pass run
             output_model.model_attributes = output_model.model_attributes or model.model_attributes
-            Pass._carry_forward_additional_files(model, output_model)
+            if not isinstance(output_model, CompositeModelHandler):
+                # not sure if we should carry forward additional files for composite model
+                # if we should, should we carry forward for each component or just the composite model
+                Pass._carry_forward_additional_files(model, output_model)
```

Comment:
What are the additional files?

Reply:
these are files like tokenizer files that get saved with the exported model so that it can be run with genai config. this logic carries forward these files with the output model.

Reply:
We would need to carry forward these files for composite model also. How would we handle tokenizer files after SplitModel pass is run?

Reply:
It gets carried forward using the `additional_files` attribute in the composite model handler's `model_attribute`. 
![image](https://github.com/user-attachments/assets/88d2d017-4316-45ad-9906-9ac5e196aa7a)

I will update the comment to make it clear. the comment was before I made some changes to the code to make this work.

Reply:
ok. thanks!

Reply:
done